articles/100_ament.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: The meta build system "ament"
+permalink: articles/ament.html
+abstract:
+  This article describes the meta build system "ament".
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## ament
+
+`ament` is a meta build system to build applications which are split into separate packages.
+
+It relies on meta information about the packages to determine their build dependencies.
+These meta information is specified in a file called `package.xml` which is defined in [REP 140](http://www.ros.org/reps/rep-0140.html).
```

Comment:
I would say "This meta information is" but if you want to use these I'd say "These meta information are".